cibyl/sources/zuul/queries/composition/factory.py

```diff
@@ -14,20 +14,9 @@
 #    under the License.
 """
 from cibyl.sources.zuul.apis import ZuulAPI as Zuul
-from cibyl.sources.zuul.managers import SourceManager
-from cibyl.sources.zuul.managers.verbose import VerboseManager
+from cibyl.sources.zuul.queries.composition.quick import QuickQuery
 
 
-class SourceManagerFactory:
-    """Factory for :class:`SourceManager`.
-    """
-
-    def from_kwargs(self, api: Zuul, **kwargs) -> SourceManager:
-        """Chooses the manager type from the undefined arguments passed to
-        this.
-
-        :param api: Interface to communicate with the Zuul host.
-        :param kwargs: Arguments coming from the CLI.
-        :return: A new manager instance.
-        """
-        return VerboseManager(api)
+class AggregatedQueryFactory:
+    def from_kwargs(self, api: Zuul, **kwargs):
+        return QuickQuery(api)
```

Comment:
VerboseQuery is currently not used, right? What will be the criteria to use QuickQuery or VerboseQuery? The '--verbose' argument?

Reply:
I will use VerboseQuery as the default choice for now, just as I have been doing until now. We still need to decide on the condition that will switch between the two.